src/xunit.performance.core/xunit.performance.core.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <AssemblyTitle>xunit.performance.core</AssemblyTitle>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
nit: I think this could stay as `netstandard1.3`

Reply:
netstandard2.0 pull less dependencies in which results in a cleaner package graph.